lib/ansible/modules/windows/win_find.ps1

```diff
@@ -172,6 +173,31 @@ Function Assert-Pattern($info) {
     $valid_match
 }
 
+Function Assert-Contains($info) {
+    $valid_match = $false
+
+    if ($contains -ne $null) {
```

Comment:
`$null -ne $contains` is the recommended way to check this for lists. This is because `$contains -ne $null` will be true if any value in `$contains` is null which isn't what you want.